Services/BookService.cs

```diff
@@ -52,7 +52,8 @@ public List<Book> GetBookByCategory(int categoryid)
             }
             catch (Exception ex)
             {
-                throw ex;
+                //throw ex;
+                return null;
```

Comment:
any specific reason for returning null in catch instead of throwing ex?

Reply:
reason for returning null is to check in front end if result is null show error message.

Reply:
Send proper message instead of null